js/26_09_19.test.js

```diff
@@ -0,0 +1,13 @@
+const likes = require('./26_09_19');
+
+
+describe('26_09_19', () => {
```

Comment:
Man oh man, now you can locate the file easily, but you do not know what is being tested unless you actually open up the file :smile: I guess it would be best if you used the file name as well as the function name, for example. Or some other way. Figure it out next time :smile: